editor/src/components/editor/global-shortcuts.tsx

```diff
@@ -903,7 +917,48 @@ export function handleKeyDown(
               return []
             }
 
+            const parentBounds =
```

Comment:
I think this entire piece of code (everything under `REMOVE_ABSOLUTE_POSITIONING`) needs to be extracted into its own file plus helper functions